src/CommandLine/NServiceBus.Transport.AzureServiceBus.CommandLine.csproj

```diff
@@ -5,7 +5,7 @@
     <OutputType>Exe</OutputType>
     <ToolCommandName>asb-transport</ToolCommandName>
     <PackAsTool>True</PackAsTool>
-    <Description>.NET Core global tool to generate topology for NServiceBus endpoints running on Azure Service Bus</Description>
+    <Description>.NET Core global tool to manage deployment artifacts like queues and topics for NServiceBus endpoints running on Azure Service Bus</Description>
```

Comment:
I'd be ok with "manage topology", but not "deployment artifacts..."
:-1: for using "like"

Reply:
>.NET Core global tool to manage deployment artifacts like queues and topics for NServiceBus endpoints running on Azure Service Bus

How about

>.NET Core global tool to deploy, update, and remove Azure Service Bus entities for NServiceBus endpoints.

Reply:
Take my money!

Reply:
I like @boblangley's suggestion!

Would "create" be better than "deploy" ?

Reply:
> Would "create" be better than "deploy" ?

I'd stick to "manage". You can create and delete.

Reply:
👍 for

> .NET Core global tool to manage Azure Service Bus entities for NServiceBus endpoints.

Reply:
Like it, PR updated :shipit: